pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -2047,15 +2043,7 @@ func GracePeriodSeconds(vmi *v1.VirtualMachineInstance) int64 {
 }
 
 func InterpretTransitionalModelType(useVirtioTransitional *bool, archString string) string {
-	// "virtio-non-transitional" and "virtio-transitional" are both PCI devices, so they don't work on s390x.
-	// "virtio" is a generic model that can be either PCI or CCW depending on the machine type
-	if isS390X(archString) {
-		return "virtio"
-	}
-	if useVirtioTransitional != nil && *useVirtioTransitional {
-		return "virtio-transitional"
-	}
-	return "virtio-non-transitional"
+	return NewArchConverter(archString).transitionalModelType(useVirtioTransitional)
```

Comment:
nit: feels a bit weird that `transitionalModelType()` gets a pointer to a bool. I think it would have been a bit cleaner to check if this pointer is nil here, then pass a non-pointer bool to `transitionalModelType()`.

This is a very minor issue though. Feel free to ignore.

Reply:
Done!